augur/cli/db.py

```diff
@@ -331,3 +332,188 @@ def check_pgpass_credentials(config):
         else:
             logger.info("Credentials found in $HOME/.pgpass")
 
+@cli.command('update-org-repositories')
+@click.argument('org')
+@pass_application
+def update_org_repositories(augur_app, org):
+    """
+    Add new repositories for specific organization to Augur's database
+    """
+
+    repos = get_all_repos(org)
+    repo_group_id = get_org_id(augur_app, org)
+    existing_repos = get_existing_repos(augur_app, repo_group_id)
+
+    for repo in repos:
+        if repo['node']['name'] not in existing_repos:
+            insert_repo(augur_app, repo['node'], org, repo_group_id)
+
+def generate_repo_id(augur_app):
```

Comment:
There is a sequence in the database that automatically increments the repoID, and this is the preferred way to do incrementing of repo_id. 

This is a **very** weird SQL way for a relational database to work.  And Postgres does a poor job of making this clear.